modules/cpmso-yaml-failure-domain-openstack.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-configuration.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-failure-domain-openstack_{context}"]
+= Sample OpenStack failure domain configuration
+
+The control plane machine set concept of a failure domain is analogous to existing OpenStack concept of an link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-availability-zones[_Availability Zone (AZ)_]. The `ControlPlaneMachineSet` CR spreads control plane machines across multiple failure domains when possible. 
```

Comment:
why do you put a link of AWS in the OpenStack section?

In Jira, I gave you a few examples; [please check them out again](https://issues.redhat.com/browse/OSASINFRA-3100?focusedId=22844819&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22844819).

Reply:
My apologies. Let me change it.